packages/@react-stately/table/src/utils.ts

```diff
@@ -0,0 +1,116 @@
+import {GridNode} from '@react-types/grid';
+import {Key} from 'react';
+
+type mappedColumn<T> = GridNode<T> & {
+    index: number,
+    delta: number,
+    calculatedWidth?: number
+};
+
+export function mapToArray(columnWidthsMap: Map<Key, number>): { key: Key, width: number }[] {
+  return Array.from(columnWidthsMap, ([key, width]) => ({key, width}));
+}
+    
+export function getContentWidth(widths) {
+  return Array.from(widths).map(e => e[1]).reduce((acc, cur) => acc + cur, 0);
+}
+    
+export function isStatic(width: number | string): boolean {
+  return width !== null && width !== undefined && (typeof width === 'number' || width.match(/^(\d+)%$/) !== null);
```

Comment:
> I think I'm interpreting it as *fr/null/undefined are dynamic columns and numbers or percents are static?

Yes that is the case and I think our documentation is out of date because we used to say that percents were dynamic, when that is not true anymore.

Reply:
btw I just realized why you did `width != undefined` instead of `width !== undefined` (I previously thought it was an accident) and so I went to make that change but it looks like the es-lint rule doesn't like that. So we should probably keep it explicitly `width !== null && width !== undefined` ? 

![image](https://user-images.githubusercontent.com/6720502/157776284-a1218242-756c-404a-a377-81b48360e04a.png)

Reply:
ah, sorry, `width != null` should pass lint then, I see we have that in a number of places in our code base, I never remember it off the top of my head, sorry for the confusion